File: src/main/java/com/es2/vadebicicleta/externo/email/model/RequisicaoEmail.java
package com.es2.vadebicicleta.externo.email.model;

import jakarta.persistence.*;
import lombok.AllArgsConstructor;
import lombok.Builder;
import lombok.Data;
import lombok.NoArgsConstructor;

@Data
@Builder
@AllArgsConstructor
@NoArgsConstructor
@Entity
public class RequisicaoEmail {
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private Long id;

    @Column
    private String email;

    @Column
    private String assunto;

    @Column
    private String mensagem;


}

File: src/main/kotlin/com/es2/vadebicicleta/externo/email/model/RequisicaoEmail.kt
package com.es2.vadebicicleta.externo.email.model

import jakarta.persistence.*

@Entity
class RequisicaoEmail (
    @Column(nullable = false)
    var email: String,

    @Column(nullable = false)
    var assunto: String,

    @Column(nullable = false)
    var mensagem: String,

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var id: Long? = null,
)
